test/helpers/time.js

```diff
@@ -21,8 +21,8 @@ module.exports = {
 
 // TODO: deprecate the old version in favor of this one
 module.exports.bigint = {
-  clock: mapObject(module.exports.clock, fn => () => fn().then(BigInt)),
-  clockFromReceipt: mapObject(module.exports.clockFromReceipt, fn => receipt => fn(receipt).then(BigInt)),
+  clock: mapObject(module.exports.clock, fn => () => fn().then(ethers.toBigInt)),
+  clockFromReceipt: mapObject(module.exports.clockFromReceipt, fn => receipt => fn(receipt).then(ethers.toBigInt)),
   forward: module.exports.forward,
-  duration: mapObject(module.exports.duration, fn => n => BigInt(fn(n))),
+  duration: mapObject(module.exports.duration, fn => n => ethers.toBigInt(fn(ethers.toNumber(n)))),
```

Comment:
Is there any reason for preferring `ethers.toBigInt` `BigInt`? I see `fn` is returning is the same anyways so there shouldn't be a diff.

Reply:
I tink ethers's version does more check that the value provided is correct. Also it clearly states that it covers a varriety of input types. 

`ethers.toBigInt(value: BigNumberish | Uint8Array)⇒bigint`

I also think the name is more explicitely saying that a conversion is hapening.

In our case, both may work just fine.